test/ServiceBus.Tests/EvictionStrategyTests/EHPurgeLogicTests.cs

```diff
@@ -233,7 +230,15 @@ private Task AddDataIntoCache(EventHubQueueCacheForTesting cache, int count)
                 count--;
                 //just to make compiler happy
                 byte[] ignore = { 12, 23 };
-                cache.Add(new EventData(ignore), DateTime.UtcNow);
+                var eventData = new EventData(ignore);
+                DateTime now = DateTime.UtcNow;
```

Comment:
'now' is not guaranteed to be unique, does it need to be?

Reply:
no. I added those to avoid NullReferenceException when runtime try to serialize those missing fields, EnqueueTime and such. 
In full dotnet, those fields would be filled out by default by just calling new EventData(ignore). But in netstandard, it is not the case anymore. 

So I need to manually assign the values. But how much is the value doesn't affect tests

Reply:
Fair enough.  Since our purge logic was time based, I suspected the timestamps may have specific requirements.  NM :)